BDBOAuth1Manager/BDBOAuth1RequestSerializer.m

```diff
@@ -287,7 +307,12 @@ - (NSDictionary *)OAuthParameters {
     parameters[BDBOAuth1SignatureVersionParameter]     = @"1.0";
     parameters[BDBOAuth1SignatureConsumerKeyParameter] = self.consumerKey;
     parameters[BDBOAuth1SignatureTimestampParameter]   = [@(floor([[NSDate date] timeIntervalSince1970])) stringValue];
-    parameters[BDBOAuth1SignatureMethodParameter]      = @"HMAC-SHA1";
+
+    if (self.RSAPrivateKey) {
```

Comment:
I forget the truth table in Objective-C--this will be true if the `RSAPrivateKey` is initialized and not `0`?